content/03-how-tos/06-How-to-contribute-to-the-Mantis-roadmap.md

```diff
@@ -0,0 +1,25 @@
+---
+title: How to Contribute to the Mantis Roadmap
+metaTitle: How to contribute to the Mantis roadmap
+---
+# How to Contribute to the Mantis Roadmap
+
+## Introduction
+
+The Mantis team at IOHK is hard at work developing exciting new features for the Mantis client, and you are welcome to add your contribution.
+
+## Ways you can help
+
+The Mantis team welcomes offers of help. 
+
+Writing code is valuable, but new code can be problematic to integrate into the existing code base.
+
+There is much more to creating new features than coding. The developers welcome contributions in these areas:
+- Ideas for improvement. The best ideas contribute something specific to benefit all the people who use the Mantis wallet.
+- Critiquing improvement candidates. Your knowledge and experience can help everyone, including yourself.
+- Reviewing and suggesting improvements to code. A new set of eyes can often see something that the team has missed or could be improved.
+- Improving documentation. Perhaps you find a topic that is not clear. Perhaps you see that annoying typo. The developers would love to hear from you.
+
+## Get in touch
+
+To discuss this in detail, contact us at mantis@iohk.io.
```

Comment:
It might be nice to lower the barrier to entry a bit. There's some easier alternatives here we can propose like creating an issue or joining Discord.